src/main/java/org/sonar/plugins/stash/StashIssueReportingPostJob.java

```diff
@@ -1,44 +1,45 @@
 package org.sonar.plugins.stash;
 
-import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.sonar.api.BatchComponent;
 import org.sonar.api.batch.PostJob;
 import org.sonar.api.batch.SensorContext;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.issue.Issue;
 import org.sonar.api.issue.ProjectIssues;
 import org.sonar.api.resources.Project;
-import org.sonar.plugins.stash.client.SonarQubeClient;
 import org.sonar.plugins.stash.client.StashClient;
 import org.sonar.plugins.stash.client.StashCredentials;
 import org.sonar.plugins.stash.exceptions.StashConfigurationException;
-import org.sonar.plugins.stash.issue.CoverageIssuesReport;
-import org.sonar.plugins.stash.issue.SonarQubeIssuesReport;
 import org.sonar.plugins.stash.issue.StashDiffReport;
 import org.sonar.plugins.stash.issue.StashUser;
 
-public class StashIssueReportingPostJob implements PostJob {
+import java.util.List;
+
+public class StashIssueReportingPostJob implements PostJob, BatchComponent {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(StashIssueReportingPostJob.class);
   private static final String STACK_TRACE = "Exception stack trace";
   
   private final ProjectIssues projectIssues;
   private final StashPluginConfiguration config;
-  private final InputFileCache inputFileCache;
   private final StashRequestFacade stashRequestFacade;
-  private final InputFileCacheSensor inputFileCacheSensor;
-    
+  private final InputFileCache inputFileCache;
+  private final FileSystem fileSystem;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "fileSystem" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)